src/mudlet.cpp

```diff
@@ -256,8 +255,8 @@ mudlet::mudlet()
     actionVars->setToolTip(tr("Show and edit lua variables"));
     mpMainToolBar->addAction( actionVars );
 
-    QAction * actionIRC = new QAction( QIcon( QStringLiteral( ":/icons/internet-telephony.png" ) ), tr("Help Chat"), this);
-    actionIRC->setToolTip(tr("Join Mudlet help chat on IRC"));
+    QAction * actionIRC = new QAction( QIcon( QStringLiteral( ":/icons/internet-telephony.png" ) ), tr("IRC"), this);
+    actionIRC->setToolTip(tr("Open the Mudlet IRC client."));
```

Comment:
Be aware _I think_ there are some dead/redundant/unused/duplicate `QActions` in this constructor (and the `menu` of the `main_window.ui` dialog/form file)... but as it IS added to the `mpMainToolBar` in the next line I guess this isn't such a one...!

Ah, yeah, see the `mActionXXXX` ones further down!